### DISCUSSION TITLE
Load the last-used memory page (MCR file) at power on.

### DISCUSSION THREAD

**Discussion by ScooberDiver** [dangiu/PicoMemcard#88] [Ideas]
Hi, it would be very useful if the unit would remember which .MCR file you selected at the last session and then at power-up later, load that same MCR file again.

Maybe when changing pages (MCR files) write the MCR filename to a config file or something like LAST.TXT ?

Thank you

**Comment by whitezombie2000** [comment]
There is a fork that does this. https://github.com/realJoshByrnes/PicoMemcard
